### PR TITLE
Font: Add Ollama logo

### PR DESCRIPTION
**Pull request by abeatrix**
PR to add the Ollama logo to the Cody font package.

Example image when replacing openai-logo with ollama-logo:

image

Related to https://github.com/sourcegraph/cody/pull/3282

## Test plan

Follow instruction here: https://github.com/sourcegraph/cody/pull/3281#discussion_r1513142477

### CODE REVIEW

File: vscode/src/edit/input/get-items/model.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ const getModelProviderIcon = (provider: string): string => {
             return '$(openai-logo)'
```
Comment (abeatrix) [inline vscode/src/edit/input/get-items/model.ts:13]: I replaced this with `ollama-logo` but it's not showing the correct logo. @umpox do you know what could be the case here?
image

Comment (abeatrix) [reply vscode/src/edit/input/get-items/model.ts:13]: OHHH thank you! I will update the README to document this if this works, thank you!

Comment (umpox) [reply vscode/src/edit/input/get-items/model.ts:13]: also THIS IS A COOL PR 😁

Comment (abeatrix) [reply vscode/src/edit/input/get-items/model.ts:13]: For adding a logo? 😆 

And thank you!! It works!!
image

**Review by sqs — APPROVED** [review]

**Pull request merged**